modules/localgov_workflows_notifications/src/Form/LocalgovWorkflowsNotificationsSettingsForm.php

```diff
@@ -27,12 +28,12 @@ final class LocalgovWorkflowsNotificationsSettingsForm extends ConfigFormBase {
    *   The factory for configuration objects.
    * @param \Drupal\localgov_workflows_notifications\NotificationTimer $notification_timer
    *   The notification timer service.
-   * @param \Drupal\Core\Config\TypedConfigManagerInterface|null $typedConfigManager
+   * @param \Drupal\Core\Config\TypedConfigManagerInterface $typed_config_manager
    *   The typed config manager.
    */
-  public function __construct(ConfigFactoryInterface $config_factory, NotificationTimer $notification_timer, protected $typedConfigManager = NULL) {
+  public function __construct(ConfigFactoryInterface $config_factory, NotificationTimer $notification_timer, protected TypedConfigManagerInterface $typed_config_manager) {
```

Comment:
This would be adding it into `$this->typed_config_manager` which I don't think is correct.

If we're going to pass it into the parent (which is putting it into `$this->typedConfigManager`) we should probably remove the `protected` here?

Reply:
I've pushed this change.